app/controllers/groups/export.js

```diff
@@ -88,6 +88,9 @@ export default Controller.extend(FileSaverMixin, {
     }, {
       value: 'emergency_number',
       label: 'Noodnummer'
+    }, {
+      value: 'avatar_url',
+      label: 'Avatar URL'
```

Comment:
Can you change the label to profielfoto url? We always call it profielfoto to the user

Reply:
oh ja klinkt wel logisch